projects/bp-gallery/src/hooks/manage-collection-pictures.hook.ts

```diff
@@ -47,7 +47,7 @@ const useManageCollectionPictures = () => {
   );
 
   const removePictures = useCallback(
-    data => {
+    (data: any) => {
```

Comment:
Maybe we can get a better type than `any` here?

Reply:
the whole method is built with any for some reason. I didn't want to rewrite the whole method, since this is just supposed to be a vite branch, so I just fixed the typescript error that wasn't there before for some reason.

![image](https://user-images.githubusercontent.com/64468276/213169733-484e5bad-c99c-4c99-ad2a-742a59d99e79.png)